pkg/store/store_test.go

```diff
@@ -315,35 +315,47 @@ func TestList(t *testing.T) {
 
 	// List the objects with relation filter and scope Source and Target
 	objects, err = store.List(context.TODO(), &topoapi.Filters{
-		RelationFilter: &topoapi.RelationFilter{SrcId: "1234", RelationKind: "e2-node-cell", TargetKind: "", Scope: topoapi.RelationFilterScope_SOURCE_AND_TARGET},
+		RelationFilter: &topoapi.RelationFilter{SrcId: "1234", RelationKind: "e2-node-cell", TargetKind: "", Scope: topoapi.RelationFilterScope_SOURCE_AND_TARGETS},
 	})
 	assert.NoError(t, err)
 	assert.Len(t, objects, 3) // 1 node, 2 cells
 
 	objects, err = store.List(context.TODO(), &topoapi.Filters{
-		RelationFilter: &topoapi.RelationFilter{SrcId: "87893172902445058", RelationKind: "e2-cell-neighbor", TargetKind: "", Scope: topoapi.RelationFilterScope_SOURCE_AND_TARGET},
+		RelationFilter: &topoapi.RelationFilter{SrcId: "87893172902445058", RelationKind: "e2-cell-neighbor", TargetKind: "", Scope: topoapi.RelationFilterScope_SOURCE_AND_TARGETS},
 	})
 	assert.NoError(t, err)
 	assert.Len(t, objects, 3) // 58 and neighbors 57 and 59
 
 	objects, err = store.List(context.TODO(), &topoapi.Filters{
-		RelationFilter: &topoapi.RelationFilter{SrcId: "87893172902445058", RelationKind: "e2-cell-neighbor", TargetKind: "", Scope: topoapi.RelationFilterScope_TARGET_ONLY},
+		RelationFilter: &topoapi.RelationFilter{SrcId: "87893172902445058", RelationKind: "e2-cell-neighbor", TargetKind: "", Scope: topoapi.RelationFilterScope_TARGETS_ONLY},
 	})
 	assert.NoError(t, err)
 	assert.Len(t, objects, 2) // neighbors 57 and 59
 
 	objects, err = store.List(context.TODO(), &topoapi.Filters{
-		RelationFilter: &topoapi.RelationFilter{TargetId: "87893172902445058", RelationKind: "e2-cell-neighbor", TargetKind: "", Scope: topoapi.RelationFilterScope_TARGET_ONLY},
+		RelationFilter: &topoapi.RelationFilter{TargetId: "87893172902445058", RelationKind: "e2-cell-neighbor", TargetKind: "", Scope: topoapi.RelationFilterScope_TARGETS_ONLY},
 	})
 	assert.NoError(t, err)
 	assert.Len(t, objects, 2) // neighbors 57 and 59
 
 	objects, err = store.List(context.TODO(), &topoapi.Filters{
-		RelationFilter: &topoapi.RelationFilter{TargetId: "87893172902445058", RelationKind: "e2-cell-neighbor", TargetKind: "e2-cell", Scope: topoapi.RelationFilterScope_TARGET_ONLY},
+		RelationFilter: &topoapi.RelationFilter{TargetId: "87893172902445058", RelationKind: "e2-cell-neighbor", TargetKind: "e2-cell", Scope: topoapi.RelationFilterScope_TARGETS_ONLY},
 	})
 	assert.NoError(t, err)
 	assert.Len(t, objects, 2) // neighbors 57 and 59
 
+	objects, err = store.List(context.TODO(), &topoapi.Filters{
+		RelationFilter: &topoapi.RelationFilter{TargetId: "87893172902445058", RelationKind: "e2-cell-neighbor", TargetKind: "e2-cell", Scope: topoapi.RelationFilterScope_RELATIONS_ONLY},
```

Comment:
Perhaps one unti test for getting relations using src ID if it is possible.

Reply:
Good idea. Done.